rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/Negotiation.cpp

```diff
@@ -949,6 +993,56 @@ class Negotiation::Implementation
 
     return std::make_shared<Handle>(for_participant, negotiators);
   }
+
+  void set_retained_history_count(uint count)
+  {
+    retained_history_count = count;
+  }
+
+  TableViewPtr table_view(
```

Comment:
This Implementation member function will also need to be made `const` when `Negotiation::table_view(~)` is made `const`.

Reply:
fixed a7c6ccb